File: app/src/main/java/com/mytlogos/enterprise/viewmodel/ExternalUserViewModel.java
package com.mytlogos.enterprise.viewmodel;

import android.app.Application;

import androidx.lifecycle.LiveData;
import androidx.paging.PagedList;

import com.mytlogos.enterprise.model.ExternalUser;

public class ExternalUserViewModel extends RepoViewModel {

    ExternalUserViewModel(Application application) {
        super(application);
    }

    public LiveData<PagedList<ExternalUser>> getExternalUser() {
        return this.repository.getExternalUser();
    }
}

File: app/src/main/java/com/mytlogos/enterprise/viewmodel/ExternalUserViewModel.kt
package com.mytlogos.enterprise.viewmodel

import android.app.Application
import androidx.paging.PagingData
import com.mytlogos.enterprise.model.ExternalUser
import kotlinx.coroutines.flow.Flow

class ExternalUserViewModel(application: Application) : RepoViewModel(application) {
    val externalUser: Flow<PagingData<ExternalUser>>
        get() = repository.externalUser
}